platform/osx/os_osx.h

```diff
@@ -34,6 +34,7 @@
 #include "crash_handler_osx.h"
 #include "drivers/coreaudio/audio_driver_coreaudio.h"
 #include "drivers/unix/os_unix.h"
+#include "editor_settings.h"
```

Comment:
It's not needed in the header, please include it only in the cpp.

Reply:
I was already wondering when I'm supposed to put it in h and when in the cpp.
put it there and it worked so I dint bother anymore.

Reply:
Basically if it's needed in the cpp, include in the cpp. If it's needed in the header, include in the header. If it's needed in both, include in the header, since it's included in the cpp too.